torch/_dynamo/funcname_cache.py

```diff
@@ -0,0 +1,53 @@
+import tokenize
+
+cache = {}
+
+
+def clearcache():
+    cache.clear()
+
+
+def _add_file(filename):
+    try:
+        with open(filename) as f:
+            tokens = list(tokenize.generate_tokens(f.readline))
+    except Exception:
```

Comment:
```suggestion
    except IOError:
```
Do you really want syntax or tokenize generating errors caught too?